### PR TITLE
Add Indonesian Language

### PR DESCRIPTION
**Pull request by danarn17**
#149

### CODE REVIEW

**Comment by katherinepeterson** [comment]
@danarn17 I believe someone already opened a PR for this 
#122

**Comment by danarn17** [comment]
> @danarn17 I believe someone already opened a PR for this
> #122

sorry, I just realize that after open this PR.

I will resolve the conflict. Of course, it is up to you if you would merge this or not. :)

File: components/SectionsColumn.js [hunk 1]
```diff
@@ -93,6 +93,7 @@ export const SectionsColumn = ({
     localStorage.setItem('current-focused-slug', 'noEdit')
   }
 
+
```
Comment (octokatherine) [inline components/SectionsColumn.js:96]:
```suggestion
```

**Review by octokatherine — CHANGES_REQUESTED** [review]
Can you resolve conflicts and remove the extra newline added? Thanks!

**Comment by danarn17** [comment]
> Can you resolve conflicts and remove the extra newline added? Thanks!

done

**Pull request merged**